go/protocol/keybase1/notify_fs_request.go

```diff
@@ -12,8 +12,13 @@ type FSEditListRequestArg struct {
 	Req FSEditListRequest `codec:"req" json:"req"`
 }
 
+type FSSyncStatusRequestArg struct {
+	Req FSSyncStatusRequest `codec:"req" json:"req"`
+}
+
 type NotifyFSRequestInterface interface {
 	FSEditListRequest(context.Context, FSEditListRequest) error
```

Comment:
Do we want to change `FSEditListRequest` to `FSRegisterForSyncStatus` or something that would be clearer that these requests go together?